Dev docs 2929 added edits (#1348)
bigcommerce/dev-docs: docs/bigcommerce-for-wordpress/setup/multi-site.md

<div class="otp" id="no-index">

### On This Page
- [Getting your API credentials](#getting-your-api-credentials)
- [Setting up a WordPress site using API account credentials](#setting-up-a-wordpress-site-using-api-account-credentials)
- [Additional resources](#additional-resources)

</div>

When connecting more than one WordPress site to your BigCommerce store, you need to use an API account to link them. If you try to connect using the 'connect your store' flow, which uses a BigCommerce app to streamline the connection, your first WordPress site will lose its connection to BigCommerce.

### Multi-site and subdirectories
Multiple sites can share the same API credentials, or you can choose to create a new set of credentials for each site.

> | **Configuration Method** | **Is Supported** |

Author: @hatertron3000 Hey again Patrick --here's that table we discussed. Does this look alright?

Reviewer: The table itself looks good. The branding is off in the link to the docs below (BigCommerce, not Big Commerce)

Author: Stellar catch! I'll get that cleaned up and then merged.

|:-------------------------|:-----------------|
| Subdirectories           | No               |
| Subdomains               | Yes              |
| Separate Domains         | Yes*             |

> Note that embedded checkout is only supported on a single domain at a time. See the [BigCommerce for WordPress ](https://support.bigcommerce.com/s/article/BigCommerce-for-WordPress-Checkout?language=en_US#subdomain-setup) documentation.*

Reviewer 2: ```suggestion
Note that embedded checkout is only supported on a single domain at a time. See the [BigCommerce for WordPress](https://support.bigcommerce.com/s/article/BigCommerce-for-WordPress-Checkout?language=en_US#subdomain-setup) documentation.*

```

<div class="otp" id="no-index">

### On This Page
- [Getting your API credentials](#getting-your-api-credentials)
- [Setting up a WordPress site using API account credentials](#setting-up-a-wordpress-site-using-api-account-credentials)
- [Additional resources](#additional-resources)

</div>

When connecting more than one WordPress site to your BigCommerce store, you need to use an API account to link them. If you try to connect using the 'connect your store' flow, which uses a BigCommerce app to streamline the connection, your first WordPress site will lose its connection to BigCommerce.

### Multi-site and subdirectories
Multiple sites can share the same API credentials, or you can choose to create a new set of credentials for each site.

| **Configuration Method** | **Is Supported** |
|:-------------------------|:-----------------|
| Subdirectories           | No               |
| Subdomains               | Yes              |
| Separate Domains         | Yes*             |

> Note that embedded checkout is only supported on a single domain at a time. See the [Big Commerce for WordPress documentation](#bigcommerce-for-wordpress-documentation).*

Author: Resolved by adding url in place of anchor.

> 

Reviewer 2: ```suggestion
### On this page
- [Getting your API credentials](#getting-your-api-credentials)
- [Setting up a WordPress site using API account credentials](#setting-up-a-wordpress-site-using-api-account-credentials)
- [Additional resources](#additional-resources)

</div>

```

> 

Reviewer 2: Suggestion, lines 7-12 (don't know how to add to batch)
```
### On this page
- [Getting your API credentials](#getting-your-api-credentials)
- [Setting up a WordPress site using API account credentials](#setting-up-a-wordpress-site-using-api-account-credentials)
- [Additional resources](#additional-resources)

</div>
```

When connecting more than one WordPress site to your BigCommerce store, you need to use an API account to link them. If you try to connect using the 'connect your store' flow, which uses a BigCommerce app to streamline the connection, your first WordPress site will lose its connection to BigCommerce.

Multiple sites can share the same API credentials, or you can choose to create a new set of credentials for each site.

<div class="HubBlock--callout">

> <div class="CalloutBlock--">

Reviewer 2: ```suggestion
<div class="CalloutBlock--info">

```
add theme specifier to CalloutBlock class.

> 

Reviewer 2: Suggestion: line 22, add info theme to markdown for Stoplight Platform compatibility:
```
<!-- theme: info -->
```

## Getting your API credentials

> To get your store’s API credentials, log into your BigCommerce store and head to `Advanced Settings` > `API Accounts`. Then, click the blue `Create API Account` button on the top left-hand side.

Reviewer 2: ```suggestion
1. To get your store's API credentials, log into your BigCommerce store and head to `Advanced Settings` > `API Accounts`.

```
There's a list of steps here, and you've removed 1, but left 2-4.  There's also duplication b/t 1 & 2.  This suggested change works in tandem with a suggested edit to 2 below.

](//s3.amazonaws.com/user-content.stoplight.io/6116/1544044020003 "#### Click 'Create API Account' to get credentials
")

> 2. Click `Create API Account` to open up a screen that will ask you to enter a name for the API account.

Reviewer 2: ```suggestion
2. Click the blue `Create API Account` button on the top left-hand side. This opens up a screen that will ask you to enter a name and select scopes for the API account.

```
In tandem with suggested change to 1 above.

3. For the OAuth Scopes, select the following default settings:

|               **OAuth Scope**                | **Default Selection** |

> |:--------------------------------------------:|:---------------------:|

Reviewer 2: ```suggestion
|:--------------------------------------------:|:----------------------|

```

It seems more readable for the list of OAuth scopes to be left-aligned.

Reviewer 2: ```suggestion
|:--------------------------------------------:|:----------------------|

```

The list of default selections could also be left-aligned, but I don't feel as strongly about this from a readability POV.

Author: Flushed them all left because, yes, it's way easier to read.

1. To set up a WordPress site using this method, click `Enter your API credentials` on the welcome screen in the plugin. 

<!--
    title: #### WordPress Plugin Welcome Screen

    data: //s3.amazonaws.com/user-content.stoplight.io/6116/1544043727239
-->

#### WordPress plugin welcome screen
![#### WordPress Plugin Welcome Screen
](//s3.amazonaws.com/user-content.stoplight.io/6116/1544043727239 "#### WordPress Plugin Welcome Screen
")

<!--
    title: 
    data: //s3.amazonaws.com/user-content.stoplight.io/6116/1544043952871
-->

![](//s3.amazonaws.com/user-content.stoplight.io/6116/1544043952871 "")

2. Enter your API credentials on your WordPress site. Saving the API credentials on your WordPress site will direct you to name the channel that the plugin will create. This allows you to list product to the channel from within BigCommerce and link orders back to the channel that comes from the WordPress site. You can also link to an existing channel.

_Congratulations, you're done setting up your additional site!_ 

<div class="HubBlock--callout">
<div class="CalloutBlock--info">

> <div class="HubBlock-content">

Reviewer 2: ```suggestion
<div class="HubBlock-content">

```

> 1. To set up a WordPress site using this method, click `Enter your API credentials` on the welcome screen in the plugin. 

Reviewer 2: ```suggestion
1. To set up a WordPress site using this method, click the `Enter your API credentials` link near the bottom of the welcome screen in the plugin.

```

<!--
    title: #### WordPress Plugin Welcome Screen

    data: //s3.amazonaws.com/user-content.stoplight.io/6116/1544043727239
-->

#### WordPress plugin welcome screen
![#### WordPress Plugin Welcome Screen
](//s3.amazonaws.com/user-content.stoplight.io/6116/1544043727239 "#### WordPress Plugin Welcome Screen
")

> You will be asked for your store's API credentials and base path.

Reviewer 2: ```suggestion

```
see suggested change to the following step 2.

![](//s3.amazonaws.com/user-content.stoplight.io/6116/1544043952871 "")

> 2. Enter the API credentials on your WordPress site. Saving the API credentials on your WordPress site will direct you to name the channel that the plugin will create. This allows you to list product to the channel from within BigCommerce and link orders back to the channel that comes from the WordPress site. You can also link to an existing channel.

Reviewer 2: ```suggestion
2. Enter your API account credentials on the next screen. Saving the API credentials on your WordPress site will direct you to name the channel that the plugin will create. This allows you to list product to the channel from within BigCommerce and link orders back to the channel that comes from the WordPress site. You can also link to an existing channel.

```

In tandem with preceding deletion at line 161.

_Congratulations, you're done setting up your additional site!_ 

> <div class="CalloutBlock--">

Reviewer 2: ```suggestion
<div class="HubBlock--callout">
<div class="CalloutBlock--info">

```
restore 1st of 3 nested div tags that was accidentally deleted.  Add theme specifier to CalloutBlock class.

> <div class="CalloutBlock--">

Reviewer 2: ```suggestion
<div class="HubBlock--callout">
<div class="CalloutBlock--info">

```
restore 1st of 3 nested div tags that was accidentally deleted.  Add theme specifier to CalloutBlock class.
